packages/core/test/stylable-processor.spec.ts

```diff
@@ -305,6 +306,31 @@ describe('Stylable postcss process', () => {
         expect(Object.keys(result.elements).length).to.eql(1);
     });
 
+    it('should not collect typed elements or classes in unknown functional selectors', () => {
+        const result = processSource(
+            `
+            :unknown(Unknown.unknown) {}
+            ${reservedPseudoClasses.map((name) =>
```

Comment:
casting this in a template string joins it with a comma (`,`), possibly a concern?

Reply:
good catch